I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/lucinda.rpy", line 419, in script
    play sound Wow
  File "renpy/common/000statements.rpy", line 300, in execute_play_sound
    renpy.sound.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "game/lucinda.rpy", line 419, in <module>
    play sound Wow
NameError: name 'Wow' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/lucinda.rpy", line 419, in script
    play sound Wow
  File "C:\Users\manjo\OneDrive\Documents\renpy-8.0.1-sdk\renpy\ast.py", line 2232, in execute
    self.call("execute")
  File "C:\Users\manjo\OneDrive\Documents\renpy-8.0.1-sdk\renpy\ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\manjo\OneDrive\Documents\renpy-8.0.1-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 300, in execute_play_sound
    renpy.sound.play(_audio_eval(p["file"]),
  File "renpy/common/000statements.rpy", line 32, in _audio_eval
    return eval(expr, locals=store.audio.__dict__)
  File "C:\Users\manjo\OneDrive\Documents\renpy-8.0.1-sdk\renpy\python.py", line 1080, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\manjo\OneDrive\Documents\renpy-8.0.1-sdk\renpy\python.py", line 1073, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/lucinda.rpy", line 419, in <module>
    play sound Wow
NameError: name 'Wow' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.1.22070801
Operation: SWEETHEART!!! ~A Mystreet Dating Simulator~ 1.0
Sat Aug 19 23:42:56 2023
